magento/magento2
# Added description discount  for each summary(cart, order)

### Description (*)

Added description discount  for each summary(cart, order)


### Fixed Issues (if relevant)

1. magento/magento2#3594: Discount Description does not get displayed on cart page

### Manual testing scenarios (*)

1. Create a coupon with description
2. Add product to cart and apply coupon
3. Check if discount description is displayed on summary

### Questions or comments


![image](https://user-images.githubusercontent.com/5350377/64911522-2a83eb00-d6f9-11e9-9be1-cb889a59cf21.png)
![image](https://user-images.githubusercontent.com/5350377/64911537-52734e80-d6f9-11e9-9242-2ce0fb362ac6.png)

![image](https://user-images.githubusercontent.com/5350377/64911528-3ec7e800-d6f9-11e9-824f-90dc4eb0fe5e.png)


### Contribution checklist (*)
